test/unit/test_estimator.py

```diff
@@ -129,32 +129,32 @@ def test_run_default_options(self):
         backend = get_mocked_backend()
         options_vars = [
             (
-                EstimatorOptions(resilience_level=1),  # pylint: disable=unexpected-keyword-arg
-                {"resilience_level": 1},
+                EstimatorOptions(default_shots=1024),  # pylint: disable=unexpected-keyword-arg
+                {"default_shots": 1024},
             ),
             (
                 EstimatorOptions(optimization_level=1),  # pylint: disable=unexpected-keyword-arg
                 {"transpilation": {"optimization_level": 1}},
             ),
             (
                 {
-                    "optimization_level": 1,
-                    "execution": {"shots": 100},
+                    "default_precision": 0.5,
```

Comment:
fwiw this would correspond to like 4 shots. I assume this doesn't matter in this test. Set to 0.1 for ~100 shots.